docs/mesos/DockerizedVagrant.md

```diff
@@ -0,0 +1,130 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.14.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Deploying a Vagrant Dockerized Mesos Cluster with Calico
+
+In these instructions, we will create two Centos virtual machines (a master and an agent) that run all cluster services as [Docker][docker] containers.  This speeds deployment and will prevent pesky issues like incompatible dependencies.
+
+If you would prefer to run the commands manually to better understand what is being run in the script,
+check out the [Manual Dockerized Deployment guide](DockerizedDeployment.md).
+
+On the Master machine, the script installs containers for:
+
+ * Zookeeper
+ * etcd
+ * Mesos Master
+ * Marathon (Mesos framework)
+
+On the Agent machine, we will install containers for:
+
+ * Mesos Agent
+ * Calico
+
+## Preparing Host Machine
+
+To run these instructions, you will need a host machine with:
+
+ * [VirtualBox][virtualbox] to host the Mesos master and slave virtual machines
+ * [Vagrant][vagrant] to run the script that provisions the Virtual Machines
+ * 4+ GB memory
+ * 2+ CPU
+ * 80 GB available storage space (40 GB per machine)
```

Comment:
I did a bit of research.  It depends on the image that you use.  The default size for the Centos machines is 40GB.  You can customize the VM images if you create your own or write some customized code in the Vagrantfile, but I have not tested that.